cmd/operator-sdk/build/cmd.go

```diff
@@ -126,20 +154,19 @@ func buildFunc(cmd *cobra.Command, args []string) error {
 			log.Fatalf("Failed to build operator binary: %v", err)
 		}
 	}
+	return doImageBuild("build/Dockerfile", image)
+}
 
-	image := args[0]
-
+// doImageBuild will execute the build command for the Dockerfile and image informed
```

Comment:
Nit: Maybe rephrase. Not sure what image informed means.
```suggestion
// doImageBuild will execute the build command for the given Dockerfile path and image
```